README.md

```diff
@@ -109,6 +109,31 @@ in {
 }
 ```
 
+### Integrating with Home Manager
+
+Integrating with [Home Manager](https://github.com/rycee/home-manager) can be done by adding your modules to the `imports` attribute.
+You can then configure your services like usual.
+
+```nix
+let
+  nur-no-pkgs = import (builtins.fetchTarball "https://github.com/nix-community/NUR/archive/master.tar.gz") {};
+in
+{
+  imports = lib.attrValues nur-no-pkgs.repos.moredhel.hmModules.modules;
```

Comment:
I assume you have tested this?

Reply:
This works as expected (now, thanks for the catch)